src/state/actions/userProfile/getCurrentUser.js

```diff
@@ -16,7 +16,7 @@ export const getCurrentUser = () => {
       dispatch(setCurrentUser(api.data));
       return api;
     } catch (err) {
-      throw new Error(err, setFetchError(err));
+      // throw new Error(err, setFetchError(err));
```

Comment:
What is the reason to remove the error in getCurrentUser? I believe this file will be changed with a pending PR related to Auth0